### PR TITLE
added ID. Also added more possibilities to home cage enrichments

### PR DESCRIPTION
**Pull request by saskiad**
I added an ID with the url to the schema - let me know if I did this right?

### CODE REVIEW

**Comment by jtyoung84** [comment]
TLDR: It might better to use "describedBy" instead of "id." Everything else looks fine.

I looked into this a bit more. There is no standard convention. I looked at the HumanCellAtlas schemas (for example, https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/core/protocol/protocol_core.json), and they have a field called "describedBy" (in camel case, which is what we'd want also) that they're using. I think they're using that describedBy field because there are some standards where you can have a describedBy field in link headers (https://datatracker.ietf.org/doc/html/draft-wright-json-schema-01#section-8.1). I can look into whether it's good practice to host the schemas using github urls, but for now I think it should be okay.

**Review by dyf — APPROVED** [review]

**Review by jtyoung84 — APPROVED** [review]
